assets/css/variebles.css

```diff
@@ -0,0 +1,15 @@
+:root {
+  --primary-font: "open-sans";
+  --thin-font: 300;
+  --normal-font: 400;
+  --bold-font: 600;
+
+  --background-color: #112b3c;
+  --primary-color: #f66b0e;
+  --secondery-color: #a5c9ca;
+  --border: 1px solid #f66b0e;
+
+  --xsmall: 0.5em;
+  --small: 0.9em;
+  --large: 3em;
+}
```

Comment:
beautiful :heart: one minor suggestion. You can ignore if you want to.
I suggest to use `--primary-border`.